packages/flutter/test/material/chip_test.dart

```diff
@@ -1808,6 +1808,28 @@ void main() {
     expect(find.byType(InkWell), findsOneWidget);
   });
 
+  testWidgets('Chips should use Ink instead of Container for delete icon ripple', (WidgetTester tester) async {
```

Comment:
Is there a way to test that the ink renders in the foreground without testing the implementation?

Reply:
I added a test for this now.